### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
Update Availability Notify documentation:

- Add app configuration's step-by-step;
- Add customization section.

https://github.com/vtex-apps/availability-notify/tree/availability-notify-doc-update

### CODE REVIEW

**Comment by mjaspan-vtex** [comment]
Hi Brian, this is super helpful and timely.

Josh, our sponsor from World Wide Golf has just requested a demo of this,
this very week.

The documentation additions, below, are also very helpful. The one
challenge I see, is this line being removed:

"The email template that will be used is back-in-stock"

What is the rationale for this?

Thank you,

Cello Jaspan


Customer Success Manager

**Comment by Mariana-Caetano** [comment]
> The one challenge I see, is this line being removed: "The email template that will be used is back-in-stock" What is the rationale for this? Thank you, Cello Jaspan Customer Success Manager

Hi @mjaspan-vtex Cello. Actually, this sentence still on the documentation, it is just in a callout format now:

Screenshot_129

**Comment by ebigler** [comment]
These updates look great to me. @btalma do you have any feedback before this PR is approved? Is it worth mentioning (or linking) to information about VTEX broadcaster and how that service is being used to get the inventory updates?

**Review by btalma — APPROVED** [review]

**Comment by polishq** [comment]
> These updates look great to me. @btalma do you have any feedback before this PR is approved? Is it worth mentioning (or linking) to information about VTEX broadcaster and how that service is being used to get the inventory updates?

One that might be worth mentioning is that the app needs to be installed in master in order to receive the broadcaster inventory events. We've had a few people try to test the app and thought that it wasn't working because they were testing in a workspace.

**Comment by mjaspan-vtex** [comment]
I noticed this line or instruction was removed.

"The email template that will be used is `back-in-stock`"

Are we no longer using that template? If so, which template or mechanism
has replaced it?

Thank you,

Cello Jaspan


Customer Success Manager

File: docs/README.md [hunk 1]
```diff
@@ -8,25 +8,55 @@
 
 # Availability Notify
 
-## Description
-
 `AvailabilityNotfier` is a VTEX Component that shows the availability notify form that is shown when the product isn't available.
```
Comment (carolinamenezes) [inline docs/README.md:11]:
```suggestion
The Availability Notify component is responsible for showing a subscription form when a product SKU isn't available. The form lets customers subscribe to get notified when that item gets restocked. 

The app records the notification request and monitors inventory updates. This way, once the requested SKU gets back in stock, the app will email the shoppers who asked to be notified.
```

File: docs/README.md [hunk 2]
```diff
@@ -8,25 +8,55 @@
...
-The app will record the request for notification and monitor inventory updates.  When the requested sku is back in stock, the app
-will send an email to the shopper that requested to be notified.
 
-## Usage
+The app records the request for notification and monitors inventory updates.  When the requested sku is back in stock, the app will send an email to the shopper that requested to be notified.
```
Comment (carolinamenezes) [inline docs/README.md:13]:
```suggestion
```

File: docs/README.md [hunk 3]
```diff
@@ -8,25 +8,55 @@
...
+
+## Configuration
+
+1. [Install](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-installing-an-app) the Availability Notify app in the desired VTEX account by running `vtex install vtex.availability-notify` in your terminal.
 
-To use this app, you need to import it in your store theme peer dependencies on manifest.json.
+2. Open your store’s Store Theme app directory in your code editor.
+
+3. Add the Availability Notify app to your theme’s in your store theme peer dependencies on `manifest.json`.
```
Comment (carolinamenezes) [inline docs/README.md:21]:
```suggestion
3. Open your app's `manifest.json file` and add the Availability Notify app under the `peerDependencies` field.

```

File: docs/README.md [hunk 4]
```diff
@@ -8,25 +8,55 @@
...
+ "store.product": {
+    "children": [
+      "availability-notify"
+    ]
+  },
+
+```
+
+
+>ℹ️ The email template that will be used is `back-in-stock`
```
Comment (carolinamenezes) [inline docs/README.md:40]: We should specify or link to some other document explaining what is included in this email template and how to change it.

File: docs/README.md (hunk 1 above)
Comment (carolinamenezes) [reply docs/README.md:11]: It would be nice to have an image preview of this component implemented in a store.

Comment (polishq) [inline docs/README.md:11]:
```suggestion
`AvailabilityNotifier` is a VTEX Component that provides an availability notification form that is shown when the product being viewed isn't available.
```

**Review by polishq — CHANGES_REQUESTED** [review]

**Review by polishq — DISMISSED** [review]

File: docs/README.md [hunk 5]
```diff
@@ -37,9 +41,25 @@ The app records the request for notification and monitors inventory updates.  Wh
...
+11. After, you will see the email template and its configuration. For example:
 
+
+![template-back-in-stock](https://user-images.githubusercontent.com/67270558/131547198-a4eb3f0e-5a20-4e63-9f1f-d3bb312fa621.gif)
 >ℹ️ The email template that will be used is `back-in-stock`
 
+
+Now you set up the template according to your necessities. Check out the next section, [Setting up the Back in stock template](#setting-up-the-back-in-stock-template), to learn how to do it.
+
+## Setting up the Back in stock template
```
Comment (Mariana-Caetano) [inline docs/README.md:59]: @polishq  just added this section to explain the settings of the template. Can you help with this part as well?

File: docs/README.md [hunk 6]
```diff
@@ -8,25 +8,76 @@
...
+
+7. Go to **Customer** > **Message center** > **Templates**.
+
+9. Search for the `availability-notify` component template, named **BACK IN STOCK** and click on it.
+
+11. After, you will see the email template and its configuration. For example:
+
+
+![template-back-in-stock](https://user-images.githubusercontent.com/67270558/131547198-a4eb3f0e-5a20-4e63-9f1f-d3bb312fa621.gif)
+>ℹ️ The email template that will be used is `back-in-stock`
```
Comment (carolinamenezes) [inline docs/README.md:54]:
```suggestion
```

File: docs/README.md (hunk 5 above)
Comment (polishq) [reply docs/README.md:58]: Hmm, I'm not very experienced with that. Is there any general documentation on editing email templates that we could link to?

Comment (Mariana-Caetano) [reply docs/README.md:58]: There is the [Including order variables in email template](https://help.vtex.com/en/tracks/transactional-emails--6IkJwttMw5T84mlY9RifRP/fLMUCPArCYB9vcTZEZ6bi). I'm going to take a look at it.

**Comment by hugocostadev** [comment]
Could we update the README? It's important informations to complete the implementations details

**Comment by hugocostadev** [comment]
@carolinamenezes @polishq  and @ebigler  could we merge this PR, please? Thanks

**Review by albertm805 — CHANGES_REQUESTED** [review]
Can you please rebase this PR

**Comment by Mariana-Caetano** [comment]
> Can you please rebase this PR
Conflicts resolved, @albertm805. Can you review the PR? Thanks.

**Review by carolinamenezes — APPROVED** [review]

**Comment by Mariana-Caetano** [comment]
> LGTM

@polishq and @btalma could you also take a look at it? Thanks!

**Review by btalma — APPROVED** [review]

**Pull request merged**